applications/StructuralMechanicsApplication/tests/membrane_test/Membrane_Q4_Truss_PointLoad_test_material.json

```diff
@@ -0,0 +1,37 @@
+{
+    "properties": [{
+            "model_part_name": "Parts_membrane",
+            "properties_id": 2,
+            "Material": {
+                    "name": "Material",
+                    "constitutive_law": {
+                            "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw"
+                    },
+                    "Variables": {
+                            "KratosMultiphysics.YOUNG_MODULUS": 1.00000E+002 ,
+                            "KratosMultiphysics.POISSON_RATIO": 3.00000E-001,
+                            "KratosMultiphysics.THICKNESS": 1.00000E-1,
+                            "KratosMultiphysics.StructuralMechanicsApplication.PRESTRESS_11": 100.0,
+                            "KratosMultiphysics.StructuralMechanicsApplication.PRESTRESS_22": 100.0,
+                            "KratosMultiphysics.StructuralMechanicsApplication.PRESTRESS_12": 0.0
+                    },
+                    "Tables": {}
+            }
+    },
+    {
+            "model_part_name": "Parts_truss",
+            "properties_id": 1,
+            "Material": {
+                    "name": "Material",
+                    "constitutive_law": {
+                            "name": "KratosMultiphysics.StructuralMechanicsApplication.LinearElasticPlaneStress2DLaw"
+                    },
+                    "Variables": {
+                            "KratosMultiphysics.YOUNG_MODULUS": 210e10,
```

Comment:
what you do is correct, however we changed the guidelines and it would now be better if you just write 

TRUSS_PRESTRESS_PK2

instead of including where it is defined. It will now work